src/main/java/me/iblitzkriegi/vixio/expressions/channel/builder/ExprBuilderBitrate.java

```diff
@@ -0,0 +1,66 @@
+package me.iblitzkriegi.vixio.expressions.channel.builder;
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import me.iblitzkriegi.vixio.Vixio;
+import me.iblitzkriegi.vixio.util.Util;
+import me.iblitzkriegi.vixio.util.wrapper.ChannelBuilder;
+import org.bukkit.event.Event;
+
+public class ExprBuilderBitrate extends SimplePropertyExpression<ChannelBuilder, Integer> {
+    static {
+        Vixio.getInstance().registerPropertyExpression(ExprBuilderBitrate.class, Integer.class,
+                "bitrate", "channelbuilders")
+                .setName("Bitrate of voice channel")
+                .setDesc("Get the bitrate of a channel builder. " +
+                        "The default value is 64kbps for channel builders. Rates multiplied by 1000." +
```

Comment:
Not sure how important it is to mention this when they can't provide anything in the thousands or ever be returned anything in the thousands. More of a technical thing.

Reply:
because some people will be expecting what jda returns, so its better just to document that this is the behavior